specifyweb/frontend/js_src/lib/components/FormEditor/Create.tsx

```diff
@@ -112,35 +112,56 @@ function ListViews({
   readonly views: AllTableViews['database'] | AllTableViews['disk'];
   readonly onSelect: (view: ViewDefinition) => void;
 }): JSX.Element {
-  const grouped = React.useMemo(
-    () => group(views.map((view) => [view.category, view] as const)),
-    [views]
-  );
+  const grouped = React.useMemo(() => {
+    const byDiscipline = group(
```

Comment:
btw, soon enough it would be safe to use the newly-added Object.groupBy for this rather than our custom group - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy
probably too early to do that today (without transpiling) as safari has no support yet)